Makefile

```diff
@@ -137,15 +137,19 @@ fmt: $(DEX_OPER_SRCS)
 simplify:
 	@gofmt -s -l -w $(DEX_OPER_SRCS)
 
+.PHONY: golint
+golint:
+	-@$(GO_NOMOD) get -u golang.org/x/lint/golint
+
 .PHONY: check
-check:
-	@test -z $(shell gofmt -l $(DEX_OPER_MAIN) | tee /dev/stderr) || echo "[WARN] Fix formatting issues with 'make fmt'"
+check: golint
+	@gofmt -l $(DEX_OPER_SRCS)
```

Comment:
Taking into account we already have a `make fmt` target, why don't we remove this line and add it as a dependency (ie, this target could be defined as `check: fmt golint`)?

Reply:
Fixing and merging afterwards, thanks!